src/constructorio-id.js

```diff
@@ -7,12 +7,14 @@
     var defaults = {
       user_agent: null,
       persist: true,
-      client_id_cookie_name: 'ConstructorioID_client_id',
-      session_id_cookie_name: 'ConstructorioID_session_id',
+      cookie_name_client_id: 'ConstructorioID_client_id',
```

Comment:
I think what you had previously was correct, haha. Here's the comment from the other PR:

https://github.com/Constructor-io/constructorio-identity/pull/8#discussion_r435677568

Reply:
They are inconsistent across client_id and session_id though - let me see what i can do.

Reply:
I'm not sure I prefer this, but this is what it would look like:

```
    var defaults = {
      user_agent: null,
      persist: true,
      client_id_storage_location: 'cookie',
      client_id_local_name: '_constructorio_search_client_id',
      client_id_cookie_name: 'ConstructorioID_client_id',
      session_id_storage_location: 'local'
      session_id_local_name: '_constructorio_search_session_id',
      session_data_local_name: '_constructorio_search_session',
      session_id_cookie_name: 'ConstructorioID_session_id',
      session_data_cookie_name: 'ConstructorioID_session',
      cookie_domain: null,
      cookie_days_to_live: 365,
      on_node: typeof window === 'undefined',
      session_is_new: null,
    };
```